.github/workflows/commit.yml

```diff
@@ -89,8 +89,6 @@ jobs:
         uses: arduino/setup-task@v1
       - name: Initialize the repo
         run: task -v init
-      - name: Build the goat
-        run: task -v build
```

Comment:
We only did this before so that the local goat that was available for the release pre-commit hook was up-to-date.  We should avoid that overhead altogether when just releasing; the commit it creates will be linted, and this was linted already.